implement the missing SplitAndScatterRegions method

## Development Task

I can't update kvproto for TiDB due to
```
# github.com/tikv/pd/server
../../go/pkg/mod/github.com/tikv/pd@v1.1.0-beta.0.20211029083450-e65f0c55b6ae/server/server.go:254:24: cannot use s (type *Server) as type pdpb.PDServer in argument to pdpb.RegisterPDServer:
        *Server does not implement pdpb.PDServer (missing SplitAndScatterRegions method)
```

Let me implement the interface first.

--- a/go.mod
+++ b/go.mod
@@ -29,7 +29,7 @@ require (
 	github.com/pingcap/errcode v0.3.0
 	github.com/pingcap/errors v0.11.5-0.20201126102027-b0a155152ca3
 	github.com/pingcap/failpoint v0.0.0-20200702092429-9f69995143ce
-	github.com/pingcap/kvproto v0.0.0-20211029081837-3c7bd947cf9b
+	github.com/pingcap/kvproto v0.0.0-20211109071446-a8b4d34474bc
 	github.com/pingcap/log v0.0.0-20210625125904-98ed8e2eb1c7
 	github.com/pingcap/sysutil v0.0.0-20210730114356-fcd8a63f68c5
 	github.com/pingcap/tidb-dashboard v0.0.0-20211107164327-80363dfbe884

--- a/server/grpc_service.go
+++ b/server/grpc_service.go
@@ -1555,6 +1555,11 @@ func (s *Server) SplitRegions(ctx context.Context, request *pdpb.SplitRegionsReq
 	}, nil
 }
 
+// SplitAndScatterRegions split regions by the given split keys, and scatter regions
+func (s *Server) SplitAndScatterRegions(ctx context.Context, request *pdpb.SplitAndScatterRegionsRequest) (*pdpb.SplitAndScatterRegionsResponse, error) {
+	panic("unimplemented")
+}
+
 // GetDCLocationInfo gets the dc-location info of the given dc-location from PD leader's TSO allocator manager.
 func (s *Server) GetDCLocationInfo(ctx context.Context, request *pdpb.GetDCLocationInfoRequest) (*pdpb.GetDCLocationInfoResponse, error) {
 	var err error

